src/pages/index.js

```diff
@@ -39,24 +38,36 @@ class Index extends React.Component {
 					<TitleAndMetaTags />
 					<Nav />
 					<div className="grid">
-						<div className="grid-item w-10 w-7-m push-1 home-block home-block-1 mt3">
-							<h1 className="f2 f1-m lh-title">
+						<div className="grid-item w-10 w-8-m push-1 mt3">
+							<h1 className="f2 f1-xl lh-title">
 								<span>A</span> <em className="fs-normal">user experience </em>
 								<span>consultancy creating </span>
 								<em className="fs-normal">beautiful software </em>
 								<span>for</span> <em className="fs-normal">global markets.</em>
 							</h1>
-							<ContactCard
-								className="home-contact mt3 light-grey"
-								email={email}
-								tel={tel}
-								street={street}
-								city={city}
-								zip={zip}
-							/>
+							<div className="dn flex-m">
+								{offices.map(office => (
```

Comment:
If we deconstruct here, we can tidy it up a little 

```suggestion
					{offices.map(({ name, tel, address: { street, city, zip }}) => (
```